lambdas/service/app.py

```diff
@@ -1173,14 +1204,7 @@ def add_all_results_to_cart(cart_id):
     hits, search_after = EsTd().transform_cart_item_request(entity_type, filters=filters, size=1)
     item_count = hits.total
 
-    write_params = {
-        'filters': filters,
-        'entity_type': entity_type,
-        'cart_id': cart_id,
-        'item_count': item_count,
-        'batch_size': 10000
-    }
-    token = CartItemManager().start_batch_cart_item_write(user_id, cart_id, write_params)
+    token = CartItemManager().start_batch_cart_item_write(user_id, cart_id, entity_type, filters, item_count, 10000)
```

Comment:
`write_params` seems to be passthrough to the state machine as an execution input. It makes more sense to have `CartItemManager` prepare the execution input instead of this method.